editor/src/components/inspector/inspector-common.ts

```diff
@@ -812,3 +844,32 @@ export function removeExtraPinsWhenSettingSize(
     deleteProperties('always', elementMetadata.elementPath, [styleP(frameProp)]),
   )
 }
+
+export function isFixedHugFillEqual(
```

Comment:
I appreciate this was moved from elsewhere and then updated, but this equality function desperately needs improving. It's not only tricky to follow all of the cases where we consider them to be equal, but it also means that if we add anything to the `FixedHugFill` type we are going to default to returning `true`. Maybe it would be better to use an exhaustive switch statement after determining that `a.fixedHugFill` is not `null`?

Reply:
good idea! fixed :)